lib/vmdb/config/validator.rb

```diff
@@ -144,7 +144,7 @@ def server(data)
         if keys.include?(:zone)
           unless Zone.in_my_region.find_by(:name => data.zone)
             valid = false
-            errors << [:zone, "zone, \"#{data.zone}\", invalid. Should be a vaild Zone"]
+            errors << [:zone, "zone, \"#{data.zone}\", invalid. Should be a valid Zone"]
```

Comment:
Ouch... that hurts... you even had "invalid" correct, but then botched it up on the home stretch...

Fun fact:  I actually almost typed "invaild" myself... so #kettleBlack I guess

Reply:
@NickLaMuro I'm glad I got `spelling` correct in the PR. Fun fact: there is a pr where I correct spellings, and I spelt spelling wrong!